lib/sensu-handler.rb

```diff
@@ -209,6 +209,9 @@ def filter_dependencies
         if @event['check']['dependencies'].is_a?(Array)
           @event['check']['dependencies'].each do |dependency|
             begin
+              if(dependency == "")
```

Comment:
I am not a seasoned ruby person. If that is more standard style of syntax I am all for it.